### PR TITLE
[stable/horovod] Fix the bug in overwriting and changed key

### PR DESCRIPTION
**Pull request by kurarrr**
#### Is this a new chart
No

#### What this PR does / why we need it:
Includes two fixes:
- Changed overwriting sentence from := to =
- Changed the key from not existing key to existing key
  - index .Values.resources "nvidia.com/gpu" -> index .Values.resources.requests "nvidia.com/gpu"

#### Special notes for your reviewer:
Nothing special

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@kurarrr 
Could you bump the chart version please? Thank you.

**Comment by kurarrr** [comment]
@zanhsieh 
Thank you. I bumped up.
Is this OK?

File: stable/horovod/templates/config.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {{- $workerNum := .Values.worker.number -}}
 {{- $name := include "horovod.fullname" . }}
 {{- $slots := 1 }}
-{{- if index .Values.resources "nvidia.com/gpu" }}
-{{- $slots := index .Values.resources "nvidia.com/gpu" }}
+{{- if index .Values.resources.requests "nvidia.com/gpu" }}
```
Comment (zanhsieh) [inline stable/horovod/templates/config.yaml:5]: I1007 06:26:50.183] [ERROR] templates/: render error in "horovod/templates/config.yaml": template: horovod/templates/config.yaml:4:7: executing "horovod/templates/config.yaml" at <index .Values.resour...>: error calling index: index of untyped nil

Comment (kurarrr) [reply stable/horovod/templates/config.yaml:5]: I fixed this.

**Comment by kurarrr** [comment]
Hey, is there any progress?
I think this change is necessary to run horovod in GPU clusters

**Pull request closed without merging**